.github/workflows/push.yaml

```diff
@@ -0,0 +1,50 @@
+name: push
```

Comment:
create this workflow based on https://github.com/SigNoz/signoz/blob/develop/.github/workflows/push.yaml

Reply:
Actually, I don't test this workflow gets working properly.
If you don't want to bother setting up a meta-config for github actions, revert this commit.

Reply:
yes, it makes sense. Mostly looks good.